portals/soos/config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "hasStyles": true,
+  "hasScripts": false,
+  "hideCollectionFilters": true,
+  "logo": {
+    "id": "SOOS-logo",
+    "link": "http://www.soos.aq",   
+    "title": "SOOS Metadata Search"
+  },
+  "org": "SOOS",
+  "pageTitle": "Southern Ocean Observing System",   
+  "query": {
+    "bounding_box": "-180,-90,180,-30",
```

Comment:
After pulling this down and using the portal I realized this bounding box query isn't doing anything because we don't support this query for portals. You can see the support parameters on this page under "Step 4: Configure your portal" https://wiki.earthdata.nasa.gov/display/EDSC/Portal+Creation+Guide

For any parameters that you need that aren't supported you would need to work with @mreese84 to get that support added into EDSC